eng/pipelines/runtimelab.yml

```diff
@@ -95,128 +76,6 @@ stages:
         ${{ if eq(variables.isOfficialBuild, true) }}:
           buildArgs: -s clr+libs -c $(_BuildConfig)
```

Comment:
Do you plan to depend on the official builds that get pushed to the runtimelab nuget feed? If yes, I think you will want to build `host` and `installer` as well here.

Reply:
probably yes, but I'm ok taking incremental steps.

Reply:
updated. I'll check how to consume NuGet packages once the official build is done.